homeassistant/components/vera.py

```diff
@@ -0,0 +1,116 @@
+"""
+Support for Vera devices.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/vera/
+"""
+import logging
+
+from collections import defaultdict
+from requests.exceptions import RequestException
+
+from homeassistant import bootstrap
+from homeassistant.const import (
+    ATTR_SERVICE, ATTR_DISCOVERED,
+    EVENT_HOMEASSISTANT_STOP, EVENT_PLATFORM_DISCOVERED)
+from homeassistant.loader import get_component
+
+REQUIREMENTS = ['pyvera==0.2.8']
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = 'vera'
+
+VERA_CONTROLLER = None
+
+CONF_EXCLUDE = 'exclude'
+CONF_LIGHTS = 'lights'
+
+BINARY_SENSOR = 'binary_sensor'
```

Comment:
Ther are used quite few times - in defining the DEVICE_CATEGORIES dictionary, in implementiung the light/switch switch - and in the discovery code at the end of the function.

Started to remove - but thought that wasn't an improvement - but happy to change if you feel strongly.